src/App.jsx

```diff
@@ -14,33 +16,118 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_FIELD_NAME = 'name';
+const SORT_FIELD_LENGTH = 'length';
+const ENTER_GOODS = goodsFromServer;
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+function filter(arr) {
```

Comment:
The function name 'filter' is not descriptive enough. As per the checklist, method's name should start with a verb and should describe what the function does. Consider renaming it to something like 'checkIfArraysMatch'.